avro-console-scripts/src/assembly/resources.xml

```diff
@@ -0,0 +1,17 @@
+<assembly>
+  <id>resources</id>
+  <formats>
+    <format>zip</format>
+  </formats>
+
+  <fileSets>
+    <fileSet>
+      <directory>../bin</directory>
+      <outputDirectory></outputDirectory>
+      <includes>
+        <include>kafka-avro-console-*</include>
+        <include>schema-registry-run-class</include>
```

Comment:
Do you have any jars that need to be built first? Just checking since the module doesn't list any dependencies

Reply:
I'm asking because I'm not sure what that script would be used for if we don't have any Java code

Reply:
The Docker tests for Connect depend on the avro console scripts and serializer/converter libraries. It seems like they are supposed to be part of the image. We're currently installing everything + the kitchen sink via system packages, so this will let us be a little more selective about what the images pull in.

Since they're just bash scripts, I don't think we need to depend on anything internally.